test.js

```diff
@@ -487,6 +487,14 @@ test('do not extend environment with `extendEnv` option', async t => {
 	]);
 });
 
+test('use extend environment with `extendEnv: true` option', async t => {
+	process.env.TEST = 'test';
+	const command = process.platform === 'win32' ? 'echo %TEST%' : 'echo $TEST';
+	const result = await m.shellSync(command, {env: {}, extendEnv: true});
```

Comment:
You're awaiting a synchronous method. Should be:

```suggestion
	const result = await m.shell(command, {env: {}, extendEnv: true});
```

The synchronous methods are slow, so we should prefer to use async methods in tests whenever possible.